include/sys/proc.h

```diff
@@ -60,8 +60,9 @@ typedef enum { PS_NORMAL, PS_STOPPED, PS_DYING, PS_ZOMBIE } proc_state_t;
 
 typedef enum {
   /* Cleared when continued or reported by wait4. */
-  PF_STOPPED = 0x1,   /* Set on stopping */
-  PF_CONTINUED = 0x2, /* Set when continued */
+  PF_STOPPED = 0x1,             /* Set on stopping */
```

Comment:
Can we actually merge `PF_STOPPED` and `PF_CONTINUED` into `PF_STATE_CHANGED` ?

Reply:
Yes, I think it's possible.

Reply:
I have a pending change that does that I'll commit it.